### DISCUSSION TITLE
Firmware comparison clarification

### DISCUSSION THREAD

**Discussion by Yahav** [djsime1/awesome-flipperzero#100] [General]
So, at the following page:
https://github.com/djsime1/awesome-flipperzero/blob/main/Firmwares.md
The Unleased firmware include:
Can be used to capture and send dynamic encrypted protocols/rolling codes. (Modern garage doors, etc.)
However, no other firmwares include the same item although most of them are forks of it.
Does that mean that Xtream for example doesn't include this capability?

**Comment by djsime1** [comment]
Generally speaking, any downstream forks will include features from their source firmware.
